curriculum/challenges/english/blocks/workshop-quincys-job-tips/68e96ce52087043b0e999296.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 68e96ce52087043b0e999296
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+Below the paragraph element, add a `main` element and nest three `section` elements inside it.
+
+# --hints--
+
+You should have a `main` element below the paragraph element.
+
+```js
+const mainEl = document.querySelector('p + main');
+assert.exists(mainEl);
+```
+
+Nested inside your `main` element you should have three `section` elements.
+
+```js
+const thirdSectionEl = document.querySelector('main > section:nth-of-type(3)');
+assert.exists(thirdSectionEl);
```

Comment:
using `assert.lengthOf` would be better here because it matches the approach in the other workshops